src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -41,29 +41,32 @@
 import org.xembly.Directives;
 
 /**
- * Starts Amazon EC2 instance.
+ * Starts Amazon ECS instance, that running Docker + SSHd on ECS. Instance
+ * to be configured in privileged mode. Configuration needs to be stored in S3.
```

Comment:
@mkordas this is result of #1066 description. Lets ask ticket author about that.

@original-brownbear Do I have to update JavaDoc or create new puzzle?